### PR TITLE
Remove lodash from forced resolutions

### PR DESCRIPTION
**Pull request by andrewscfc**
No Issue

**Overall change:**
No longer force lodash resolution as dependencies have now updated to be compatible with the latest version of lodash.

This fixes an issue with our force resolutions implementation where `npm ci` gave different results to `npm install`. We should consider whether remove the force resolution approach altogether or whether we need to tighten up our processes/scripts for when we need to force resolutions in future.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -87,7 +87,7 @@
         "debug": "^4.1.0",
         "gensync": "^1.0.0-beta.1",
         "json5": "^2.1.2",
-        "lodash": "^4.17.20",
+        "lodash": "^4.17.19",
```
Comment (karinathomasbbc) [inline package-lock.json:90]: Didn't 4.17.19 have the security vulnerability?

Comment (karinathomasbbc) [reply package-lock.json:90]: https://snyk.io/vuln/npm:lodash@4.17.19

**Review by karinathomasbbc — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (karinathomasbbc) [reply package-lock.json:90]: It passes the synk test so I guess it must be ok? 🤷‍♀️

Comment (andrewscfc) [reply package-lock.json:90]: That indicates that the dependency _requires_ 4.17.19 and up when you run `npm ls lodash` the output indicates lodash 4.17.20 has been resolved: 
image

**Review by ryanmccombe — APPROVED** [review]

**Pull request merged**